modules/understanding-upgrade-channels.adoc

```diff
@@ -7,89 +7,80 @@
-`openshift-install` binary for a given patch level of {product-title} always
-installs that patch level.
-
-{product-title} {product-version}, which includes the upgrade from the previous 4.3
-release, has three upgrade channels to choose from:
+= {product-title} Upgrade Channels and Releases
+
+In {product-title} 4.1, Red Hat introduced the concept of channels for
+recommending the appropriate release versions for cluster upgrade. By controlling
+the pace of upgrades, these upgrade channels allow users to choose an upgrade
+strategy. Upgrade channels are tied to a minor version of
+{product-title}. For instance, {product-title} {product-version}
+upgrade channels will never include an upgrade to a 4.5 release. This ensures
+administrators make an explicit decision to upgrade to the next minor version of
+{product-title}. Upgrade channels only control release selection and have
+no impact on the version of the cluster you install; the `openshift-install`
+binary for a given patch level of {product-title} always installs that patch level.
+
+{product-title} {product-version} has the following upgrade channels to choose from:
 
 * candidate-{product-version}
 * fast-{product-version}
 * stable-{product-version}
 
-The upgrade channels contain two types of updates:
-
-.  General Availability Software (or GA) - These versions of {product-title} are
-fully supported and are considered production quality. You may upgrade to the general
-availability release from either of the fast and stable channels.
+[discrete]
+== candidate-{product-version} Channel
 
-.  Release Candidate Software (or RC) - These versions of {product-title} are
-representative of the eventual general availability release and are available only
-in the candidate-{product-version} channel. The release candidate will contain all
-the features of the product. You are allowed to upgrade from a release candidate to
-another release candidate and to upgrade from a previous minor version of
-{product-title} to the current release candidate. Release candidate builds are not
-supported by Red Hat and you will not be able to upgrade from a release candidate to
-the general availability release of {product-title}. Candidates should be used to
+The candidate-{product-version} channel will contain candidate builds for all z-stream
+({product-version}.z) releases. If a given release is only available in the candidate-{product-version}
+channel and not available in either the fast-{product-version} or stable-{product-version}
+channels, it is considered a release candidate. Although release candidates contain
+all the features of the product, they are not supported and should only be used to
 test feature acceptance and assist in qualifying the next version of
-{product-title} in your infrastructure.
-+
+{product-title} in your infrastructure. Additionally, since a release candidate
+can only be upgraded to another release candidate, candidate-{product-version} may include
```

Comment:
/cc @LalatenduMohanty @wking This refers only to what would be available in our upgrade channels - candidate-4.x, fast-4.x, stable-4.x - and as far as I know this has not changed.

Reply:
yes, i also mean in the upgrade channels. I catch 4.3.0-rc.3 to 4.3.0 upgrade path with following command.
```
curl -sH 'Accept:application/json' 'https://api.openshift.com/api/upgrades_info/v1/graph?channel=candidate-4.3' | /home/liujia/work/cincinnati/hack/graph.sh | dot -Tpng >graph.png
```
Also, i checked it in https://github.com/openshift/cincinnati-graph-data/blob/master/channels/candidate-4.3.yaml, am i mis-understanding "a release candidate can only be upgraded to another release candidate"

Reply:
The statement `a release candidate can only be upgraded to another release candidate` is true. When builds gets releases in candidate channel, they are release candidate that point of time. However if we find it stable enough  it gets promoted to fast and stable channel which contains supported versions. I guess the current text confuses the user because of the previous sentence `If a given release is only available in the candidate-{product-version} channel and not available in either the fast-{product-version} or stable-{product-version}, it is considered a release candidate. `  Because every build when introduced in to candidate channel is a release candidate and then later we decide if we are going to support it by moving to fast and stable. let me know if that makes sense.